dbt_cloud_plugin/helpers.py

```diff
@@ -0,0 +1,44 @@
+from airflow.utils.task_group import TaskGroup
+
+from .operators.dbt_cloud_check_model_result_operator import DbtCloudCheckModelResultOperator
+
+
+def generate_dbt_model_dependency(dbt_job_task, downstream_tasks, dependent_models, ensure_models_ran=True):
```

Comment:
it might be helpful to add in some param type helpers here

Reply:
Yea, I think the types in the docstring help

Reply:
ah ok, I was thinking something similar to how you set up the snowflake to dynamo operator. but if the inputs can be multiple types and structures your approach makes sense

https://github.com/nursefly/nursefly-analytics/blob/master/airflow/nursefly_extensions/operators/snowflake_to_dynamo.py#L73